docs/connector-development/connector-builder-ui/error-handling.md

```diff
@@ -0,0 +1,84 @@
+
+## Backoff strategies
+
+The API documentation will usually cover when to reattempt a failed request that is retryable. This is often through a `429 Too Many Requests` response status code, but it can vary for different APIs. The following backoff strategies are supported in the connector builder:
+* [Constant](#constant)
+* [Exponential](#exponential)
+* [Wait time from header](#wait-time-from-header)
+* [Wait until time from header](#wait-until-time-from-header)
+
+### Constant
+
+When the API documentation recommends that requests be retried after waiting a constant amount of time, the "Constant" backoff strategy should be set on the error handler.
+
+#### Example
+
+The [Intercom API](https://developers.intercom.com/intercom-api-reference/reference/http-responses) is an API that recommends a constant backoff strategy when retrying requests.
+
+### Exponential
+
+The exponential backoff strategy is similar to constant where the connection waits to retry a request based on a numeric value defined on the connector. When the API documentation recommends that requests be retried after waiting an exponentially increasing amount of time, the "Exponential" backoff strategy should be set on the error handler.
+
+Note: When no backoff strategy is defined, the connection defaults to using an exponential backoff to retry requests.
+
+#### Example
+
+The [Delighted API](https://app.delighted.com/docs/api#rate-limits) is an API that recommends using an exponential backoff. In this case, the API documentation recommends retrying requests after 2 seconds, then 4 seconds, and so on. 
+
+Although a lot of API documentation does not call out using an exponential backoff, some APIs like the [Posthog API](https://posthog.com/docs/api) mention rate limits that are advantageous to use an exponential backoff. In this case, the rate limit of 240 requests/min should work for most syncs. However, if there is a spike in traffic, then the exponential backoff allows the connection to avoid sending more requests than the endpoint can support.
+
+### Wait time from header
+
+The "Wait time from header" backoff strategy allows the connection to wait before retrying a request based on the value specified in the API response.
+
+#### Example
+
+The [Chargebee API](https://apidocs.chargebee.com/docs/api/error-handling) documentation recommends using the `Retry-After` in the response headers to determine when to retry the request.
```

Comment:
Could you add an example here how that looks in practice? Like "this request" returns "this header", which means we need to refetch at "time"